scalableminds/webknossos
# Fix padding for 3D view buttons and update test screenshots

- Reintroduces a padding that @philippotto removed in https://github.com/scalableminds/webknossos/commit/693029f352969abe102dcb88acf98b8ef55566a4#diff-aebd91e5f092a8ae7cfa0daeff212876fa3274e7187914fc56ff96fe86258f3bL73, but multiple libraries were updated before and after, so I guess his earlier "fix" was only needed temporarily
- Updates screenshots, because the 3D view zoom somehow changed slightly after the react 18 update

### URL of deployed dev instance (used for testing):
- https://fixscreenshottests.webknossos.xyz/

### Steps to test:
- I ran the nightlies on this branch and they succeeded
- Open a dataset and check that the padding between the colored dot and the viewport name in the 3D view buttons looks good (not bunched up as before)

### TODOs:
- [x] Remove nightlies triggers again

### Issues:
- fixes nightlies
